File: instaCults/Models/CultMember.cs
namespace instaCults.Models;

public class CultMember
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public int CultId { get; set; }
    public string AccountId { get; set; }
}

File: instaCults.client/src/models/CultMember.js
export class CultMember {
    constructor(data) {
        this.id = data.id
        this.cultId = data.cultId
        this.accountId = data.accountId
        this.createdAt = new Date(data.createdAt)
        this.updatedAt = new Date(data.updatedAt)
    }
}